stdlib/cmake/modules/AddSwiftStdlib.cmake

```diff
@@ -415,13 +415,11 @@ function(_add_target_variant_link_flags)
   endif()
 
   if(NOT SWIFT_COMPILER_IS_MSVC_LIKE)
-    # FIXME: On Apple platforms, find_program needs to look for "ld64.lld"
-    find_program(LDLLD_PATH "ld.lld")
-    if((SWIFT_ENABLE_LLD_LINKER AND LDLLD_PATH AND NOT APPLE) OR
+    if((SWIFT_USE_LINKER STREQUAL "lld") OR
        ("${LFLAGS_SDK}" STREQUAL "WINDOWS" AND
         NOT "${CMAKE_SYSTEM_NAME}" STREQUAL "WINDOWS"))
       list(APPEND result "-fuse-ld=lld")
-    elseif(SWIFT_ENABLE_GOLD_LINKER AND
+    elseif(SWIFT_USE_LINKER STREQUAL "gold" AND
```

Comment:
I think the manipulations around these lines (L418-L422) are the same to the ones in `AddSwiftUnittests.cmake`, but the modifications are different: some checks are kept here, while they are drop in the other file. Is the divergence intentional?